spec/Dockerfile_spec.rb

```diff
@@ -86,7 +76,7 @@
 
               describe entrypoint_command("env") do
                 its(:stdout) { should include_env "TEST_VAR", "service-var" }
-                its(:stderr) { should be_empty }
+                its(:stderr) { should match /(^$|(apps).*(TEST_VAR).*(loaded deprecated value))/ }
```

Comment:
some of these I did try and make sure reflected /which/ of the TEST_VARs got matched. Not as important as the value so didn't do this everywhere.